styles.css

```diff
@@ -401,7 +397,62 @@ button:active {
 
 .touch h2 {
   color: #172b4d;
-  margin-top: 30px;
   font-size: 2.3em;
 }
 
+.connect {
+  display: flex;
+  width: 140%;
+  height: 80%;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 40px;
+  padding: 24px;
+}
+
+#comment {
+  background: url("/Images/form-handle.png") 100% 100% no-repeat;
+  background-size: 10%;
+}
+
+input[type="text"],
+input[type="email"] {
+  display: flex;
+  width: 100%;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  text-align: left;
+  margin-bottom: 24px;
+  padding: 5px;
+}
+
+input::placeholder {
+  font-style: italic;
+}
+
+textarea {
+  font-size: 0.9em;
+  text-align: left;
+  border: 1px solid #60c095;
+  width: 100%;
+  border-radius: 4px;
+  padding: 20px;
+}
+
+input:invalid {
+  box-shadow: 1px 8px 16px #de564130;
+}
+
+input:valid {
+  box-shadow: 1px 8px 16px #41de8330;
+}
+
+input:focus:invalid {
```

Comment:
This line is not working as intended,try using  `select:focus`for semantic purposes